### ISSUE TITLE
It is possible to add plug-in Random-Features?

### ISSUE THREAD

**Issue by dadec666** [TehCheat/PoEHelper#23]
It is possible to add plug-in Random-Features?

**Comment by TehCheat** [comment]
Yes, you (or the author of the plugin) would need to change the .NET target of the plugin to 6.0 and fix (any errors that might manifest as a result).
